lib/APPReinforcementLearning/src/ReadyState.h

```diff
@@ -96,19 +97,67 @@ class ReadyState : public IState
      */
     void setLapTime(uint32_t lapTime);
 
+    /**
+     *  Set the selected mode.
+     *
+     * @return It returns 1 if DrivingMode is selected or 2 if TrainingMode is selected.
+     */
+    uint8_t setSelectedMode();
```

Comment:
This should be a `get` instead of a `set`, according to the implementation and the description